src/package.json

```diff
@@ -47,6 +47,7 @@
     "firestore-store": "^1.0.0",
     "gsap": "^3.10.4",
     "jsonwebtoken": "^8.5.1",
+    "likecoin-wallet-connector": "git://github.com/nwingt/likecoin-wallet-connector.git#41d65d6326430f7c652e2913170070d654f4fc8f",
```

Comment:
please also add this package to [functions](https://github.com/likecoin/liker-land/blob/master/functions/package.json)